packages/voting/graphql/schema-types.js

```diff
@@ -57,18 +60,30 @@ type VotingStatsCount {
   options: [VotingOptionResult!]!
 }
 
+type VotingMembershipRequirement {
+  membershipTypeId: ID!
+  createdBefore: DateTime!
```

Comment:
I thought about that too, but because the query to decide about the eligibility of a user / numEligible is already complicated enough and it's not a use case right now, I would like to skip it.